content/tutorials/interoperability/get-started-evm-sidechain.md

```diff
@@ -0,0 +1,116 @@
+---
+html: get-started-evm-sidechain.html
+parent: evm-sidechains.html
+blurb: Get started with the EVM compatible sidechain for the XRP Ledger.
+labels:
+  - Development, Interoperability
+status: not_enabled
+---
+# Get Started with the EVM Sidechain
+
+This getting started tutorial walks you through the steps to set up your account and transfer funds using the EVM sidechain bridge. 
+
+## 1. Create an Account Using an EVM Compatible Wallet
+
+In order to interact with the network, you need to create an account in the EVM sidechain. To create and manage this account you can use any EVM compatible wallet such as MetaMask.
+
+For instructions on how to install and create an account using MetaMask, and then send and receive tokens, see [Connect MetaMask to XRPL Sidechain](connect-metamask-to-xrpl-evm-sidechain.html).
+
+## 2. Move XRP Ledger Devnet Tokens to the EVM Sidechain
+
+Before you can start interacting with the EVM blockchain, you need to transfer some tokens from the XRP Ledger Devnet to the EVM sidechain. 
+
+To obtain tokens in the XRP Ledger Devnet, go to the [XRP Faucets](xrp-testnet-faucet.html) page and click *Generate Devnet credentials* to generate a new Devnet account with some test XRP in it as shown in the following image.
+
+![Generate XRP Ledger Devnet credentials](img/evm-sidechain-xrpl-devnet-faucet.png "Generate XRP Ledger Devnet credentials")
+
+Note the address and secret associated with your Devnet address. You will need this information to set up your preferred XRP Ledger wallet. 
+
+## 3. Transfer Funds Using the EVM Sidechain Bridge
+
+Once you have your accounts setup and test fund allocated, you can use the EVM Sidechain bridge to move the test XRP tokens to the EVM Sidechain. 
+
+The EVM Sidechain bridge is a tool that allows the user to transfer funds between chains in a fast and secure way.
+
+To start using the bridge, go to [https://bridge.devnet.xrpl.org](https://bridge.devnet.xrpl.org/)
+
+### 1. Connect Both Wallets
+
+**Connect Xumm Wallet**
+
+The Xumm wallet is used to interact with the XRP Ledger Devnet chain. 
+Ensure that you have created an account on the public XRP Ledger Devnet as described in Step 1. 
+
+To connect a Xumm wallet to the bridge, go to the [EVM Sidechain bridge](https://bridge.devnet.xrpl.org) and click “Connect with Xumm Wallet”. 
+
+![Connect XUMM Wallet](img/evm-sidechain-connect-xumm-wallet.png "Connect XUMM wallet")
+
+
+**Note:** Ensure that you are connected to XRP Ledger Devnet and that the application that you are connecting with is the correct one.
+
+Follow the instructions on screen to scan the QR code using the Xumm app. The Xumm wallet app will display a confirmation page similar to the following image.
```

Comment:
The Xumm wallet app displays...

[Again, I don't recommend a screen shot. I don't think the interface in Xumm is all that complicated.]